app/scripts/compose_json_files.py

```diff
@@ -5,28 +5,46 @@
 
 
 def main():
-    in_dir = Path(sys.argv[1])
-    out_dir = Path(sys.argv[2])
-    out_dir.mkdir(parents=True, exist_ok=True)
+    ConfigPath = Path(sys.argv[1])
```

Comment:
Python naming convention would make this `config_path`.

Reply:
Have changed